src/aura/disbursementsCreate/disbursementsCreate.cmp

```diff
@@ -0,0 +1,142 @@
+<aura:component controller="disbursementsCreateController" implements="force:lightningQuickActionWithoutHeader, force:hasRecordId">
+
+    <lightning:notificationsLibrary aura:id="notifLib"/>
+
+    <!-- Init -->
+    <aura:handler name="init" value="{!this}"
+                  action="{!c.doInit}"/>
+
+    <aura:attribute type="Map" name="model" />
+    <aura:attribute type="Map" name="data"
+        default="{
+            formData: {
+                paymentCount: 1,
+                paymentTotal: 0,
+                firstDate: null,
+                intervalCount: 1,
+                intervalType: 'Month'
+            },
+            intervalTypes:null,
+            columns: [],
+            disbursements:[]}"
+    />
+
+    <div class="wrap">
+
+        <!-- Header -->
+        <h2 class="slds-text-heading_medium slds-hyphenate slds-m-bottom--large">Schedule Bulk Disbursements</h2>
+
+        <!-- Body -->
+        <div class="slds-section slds-is-open">
+            <h3 class="slds-section__title slds-theme_shade">
+                <span class="slds-truncate slds-p-horizontal_small" title="Basics">Funding Request Information</span>
+            </h3>
+            <div class="slds-section__content">
+                <div class="slds-grid slds-grid_pull-padded-medium slds-m-bottom_large">
+
+                    <div class="slds-p-horizontal_medium slds-size_1-of-2">
+                        <dl class="slds-list_horizontal slds-wrap">
+                            <dt class="slds-item_label slds-text-color_weak slds-truncate">Contact:</dt>
+                            <dd class="slds-item_detail slds-truncate">{!v.model.fundingRequest.contactName}</dd>
+                            <dt class="slds-item_label slds-text-color_weak slds-truncate">Awarded Amount:</dt>
+                            <dd class="slds-item_detail slds-truncate">
+                                <ui:outputCurrency value="{!v.model.fundingRequest.awardedAmount}"/>
```

Comment:
Any particular reason why you went with `<ui:outputCurrency>` instead of the newer `<lightning:formattedNumber>` component? (I see it a few times here, but I'm just highlighting it on this line)